src/eventcontroller/DefaultEventController.ts

```diff
@@ -38,16 +54,20 @@ export default class DefaultEventController implements EventController {
   private deviceName: string;
 
   constructor(
-    private audioVideoController: AudioVideoController,
-    private eventReporter?: EventReporter
+    configuration: MeetingSessionConfiguration,
+    logger: Logger,
+    eventReporter?: EventReporter
```

Comment:
Can we keep `private eventReporter?: EventReporter` and then use `this.eventReporter`? Do we need `private _eventReporter: EventReporter`?

Reply:
It was changed to allow for a getter. The main use of the getter outside of the EventController is testing purposes, but as we exposed the eventReporter in the MeetingSession previously it would be best to continue to expose the EventReporter inside the EventController.